src/main/steps/home/get.ts

```diff
@@ -33,18 +34,24 @@ export class HomeGetController {
       throw new Error('Invalid case type');
     }
 
-    const firstQuestionForm = getApplicantFirstQuestionForm(
-      req.session.isApplicant2,
-      req.session.userCase.applicationType!
-    );
+    const firstQuestionFormFields = req.session.isApplicant2
```

Comment:
i think it was better in the function. is there a reason it was moved out?

Reply:
The reason I moved it into getApplicant2FirstQuestionFormFields function was because then the req.session.userCase.applicationType! couldn't be undefined, where as before it could be (for app1 answering the first couple of questions)

I'm happy to change it back if you'd prefer though